src/services/Config/ConfigService.js

```diff
@@ -177,6 +177,14 @@ class ConfigService {
         return this._valueLoader.getBooleanValue('TESTRAIL_RUN_INCLUDE_ALL', 'runIncludeAll', false);
     }
 
+  /**
+     *
+     * @returns {boolean}
+     */
+    includeCasesOnlyIfExistInRun() {
+        return this._valueLoader.getBooleanValue('TESTRAIL_VALIDATE_CASE_IDS', 'validateCaseIds', false);
```

Comment:
Change this to match
```suggestion
        return this._valueLoader.getBooleanValue('TESTRAIL_IGNORE_MISSING_CASE_IDS', 'ignoreMissingCaseIds', false);
```